docs/openshift_horizontal_scaling.md

```diff
@@ -34,11 +34,12 @@ doc](./async.md).
 ### Redis databases
 
 The standard 3scale deployment uses a single Redis pod and sets both the resque
-DB and the data DB in the same Redis process under different DB indexes. This is
-not supported by the async redis library, so we need to either place both DBs in
-the same Redis process or use two separate ones. To configure this, change the
-`CONFIG_REDIS_PROXY` and `REDIS_QUEUES_URL` envs in the listener and worker
-deployment configs.
+DB and the data DB in the same Redis process under different DB indexes. If an
+external Redis is used the URLs set in `CONFIG_REDIS_PROXY` and 
+`REDIS_QUEUES_URL` can be configured to either two separate Redis instances
```

Comment:
I don't know what's `REDIS_QUEUES_URL` for, I don't think apisonator uses it. Shouldn't it be `CONFIG_QUEUES_MASTER_NAME`?.

Reply:
Yes, you're right, apisonator uses `CONFIG_QUEUES_MASTER_NAME` environment variable. I didn't check the original env vars, but there was a mix of what apisonator uses, and what operator uses :upside_down_face: 

Here is the mapping:

"backend-redis" secret | apisonator env var |
|---- | ----- |
| REDIS_STORAGE_URL | CONFIG_REDIS_PROXY |
| REDIS_STORAGE_SENTINEL_HOSTS | CONFIG_REDIS_SENTINEL_HOSTS |
| REDIS_STORAGE_SENTINEL_ROLE | CONFIG_REDIS_SENTINEL_ROLE |
| REDIS_QUEUES_URL | CONFIG_QUEUES_MASTER_NAME |
| REDIS_QUEUES_SENTINEL_HOSTS | CONFIG_QUEUES_SENTINEL_HOSTS |
| REDIS_QUEUES_SENTINEL_ROLE | CONFIG_QUEUES_SENTINEL_ROLE |

Reply:
I've amended this env var name

Reply:
I didn't know the operator uses different variables. Thanks for the info.